package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "dependencies": {
     "babel-core": "^6.9.0",
+    "babel-plugin-transform-amd-system-wrapper": "^0.1.0",
```

Comment:
There are still some references to traceur in /compilers/esm.js and in some /lib/*.js files. We have to remove them first before we can remove traceur completely.